hub.py

```diff
@@ -240,8 +240,37 @@ def get_generated_config(self, auth_provider, proxy_secret_key):
             # these can all exist fine.
             generated_config['jupyterhub']['hub']['config']['GenericOAuthenticator'] = auth_provider.get_client_creds(client, self.spec['auth0']['connection'])
 
+            # Add 2i2c staff to the list of admins and allowed users
+            admin_users, allowed_users = self.get_hub_allowed_users()
+            generated_config['jupyterhub']['hub']['config']['Authenticator'] = {
+                'admin_users': admin_users,
+                'allowed_users': allowed_users
+            }
+
         return self.apply_hub_template_fixes(generated_config, proxy_secret_key)
 
+    def get_hub_allowed_users(self):
+        # Add 2i2c staff as admins and allowed users
+        staff_emails = [
+            'yuvipanda@gmail.com',
+            'colliand@gmail.com',
+            'choldgraf@gmail.com',
+            'georgiana.dolocan@gmail.com'
+        ]
+        staff_github_ids = [
+            'yuvipanda',
+            'choldgraf',
+            'GeorgianaElena'
+        ]
+
+        extra_admins = self.spec['auth0']['extra_admins'] if self.spec['auth0'].get('extra_admins', False) else []
```

Comment:
Hmm, so we can't 'merge' this list with what's already in config, right? Is there any other way to manage this without inventing our own schema?

Reply:
Unfortunately I don't think so :( Or I just couldn't find a different working approach